lib/nats.js

```diff
@@ -118,6 +118,7 @@ const VERSION = '1.2.4',
     NO_USER_JWT_IN_CREDS_MSG = 'Can not locate user jwt in credentials.',
     BAD_OPTIONS = 'BAD_OPTIONS',
     BAD_OPTIONS_MSG = 'Options should be an object as second paramater.',
+    NO_ECHO_NOT_SUPPORTED = 'NO_ECHO_NOT_SUPPORTED',
```

Comment:
Should you use a `_MSG` for the error text?

Reply:
yeap.